Makefile

```diff
@@ -341,3 +341,13 @@ clean-operator: ## Clean Operator
 	oc delete accounts --all -n ${NAMESPACE}
 	oc delete awsfederatedaccountaccess --all -n ${NAMESPACE}
 	oc delete awsfederatedrole --all -n ${NAMESPACE}
+
+.PHONY: download
+download: ## Download dependencies
+	@echo Downloading go.mod dependencies ...
+	@go mod download
+
+.PHONY: install-tools
+install-tools: download ## Get tools
+	@echo Installing tools from tools.go ...
+	@cat tools.go | grep _ | awk -F'"' '{print $$2}' | xargs -tI % go install %
```

Comment:
Can you add a line break here?

Reply:
thanks, I already did it ;)
FYI: next time feel free to use https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request so I can click-and-commit on your proposals ;)